Using-Mastodon/FAQ.md

```diff
@@ -36,19 +36,26 @@ While Mastodon is compatible with GNU social in terms of server to server commun
 
 Because Mastodon has been created from a blank slate, it is much simpler to have the API mirror internal structures as closely as possible, rather than build an emulation layer. Secondly, the GNU social client API is actually a half-way implementation of the legacy Twitter API - that's the reason why it works with some older Twitter client apps. However, many of those apps are not maintained anymore, the GNU social API does not actually keep up with the real Twitter API and never fully implemented all its features; at the same time, the Twitter API was never meant for a federated service and so obscures some of the functionality.
 
-
 #### How is Mastodon funded?
 
 Development of Mastodon and hosting of mastodon.social is funded through my [Patreon (also BTC/PayPal donations)](https://www.patreon.com/user?u=619786). Beyond that, I am not interested in VC funding, monetizing, advertising, or anything of that sort. I could offer setup/maintenance services on demand.
 
 The software is free and open source and communities should host their own servers if they can, that way the costs are more or less distributed. Obviously it'd be hard for me to pay the bills if literally everyone decided to use the mastodon.social instance only.
 
-#### What’s a “federation”?
+#### This looks a lot like twitter, what’s the difference?
+Mastodon is a decentralized social network with no in-line ads.
```

Comment:
I think the question is more about "what makes it better?" than how to distinguish the two. It's a tough answer IMO, because most newbs aren't going to understand the benefits of decentralization. Maybe:

"Mastodon is decentralized. Anyone can run a mastodon server under their own community rules. Twitter is run by a centralized entity, and sets the rules for everyone."